assets/src/components/detours/detourMap.tsx

```diff
@@ -320,7 +320,7 @@ const WaypointMarker = ({ position }: { position: LatLngLiteral }) => (
     position={position}
     divIconSettings={{
       iconSize: [10, 10],
-      iconAnchor: new Leaflet.Point(5, 5),
+      iconAnchor: new Leaflet.Point(5, 10),
```

Comment:
I'm gonna be honest - I don't fully understand why this is necessary, and I think there's a chance that this is actually just masking (or correcting for?) an SVG error elsewhere, but I did a (-n admittedly small) bit of digging and wasn't able to figure it out.

Reply:
Based on the docs, I think we should get rid of `iconAnchor` altogether (maybe leaving a comment that it's not specified because we want it centered

> [...] Centered by default if size is specified [...]
>
> -- https://leafletjs.com/reference.html#divicon-iconanchor